src/dlgConnectionProfiles.cpp

```diff
@@ -211,6 +213,20 @@ void dlgConnectionProfiles::slot_update_discord_optin(int state)
     }
     QString profile = pItem->text();
     writeProfileData(profile, QStringLiteral("discordoptin"), QString::number(state));
+
+    // in case the user is already connected, ask for a reconnect so all
```

Comment:
comment end in middle of

Reply:
Yeah, I ran out of imagination... I also now think that avoid making the player reconnect if they enable in the middle of the session would be a better experience - so I'll put in the extra work to make it happen.